File: CS/WpfApplication48/ViewModel.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WpfApplication48
{
    public class ViewModel
    {

        private ObservableCollection<Task> _List;
        public ObservableCollection<Task> List
        {
            get
            {
                _List = new ObservableCollection<Task>();
                for (int i = 0; i < 20; i++)
                {
                    _List.Add(new Task() { Name = "Task " + i, FirstValue = i, SecondValue = i * i, IsCompleted = i % 2 != 0 });
                }
                return _List;
            }
        }

        public class Task
        {
            public string Name { get; set; }
            public int FirstValue { get; set; }
            public int SecondValue { get; set; }
            public bool IsCompleted { get; set; }
        }
    }
}

File: VB/WpfApplication48/ViewModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Collections.ObjectModel
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace WpfApplication48
	Public Class ViewModel

		Private _List As ObservableCollection(Of Task)
		Public ReadOnly Property List() As ObservableCollection(Of Task)
			Get
				_List = New ObservableCollection(Of Task)()
				For i As Integer = 0 To 19
					_List.Add(New Task() With {.Name = "Task " & i, .FirstValue = i, .SecondValue = i * i, .IsCompleted = i Mod 2 <> 0})
				Next i
				Return _List
			End Get
		End Property

		Public Class Task
			Public Property Name() As String
			Public Property FirstValue() As Integer
			Public Property SecondValue() As Integer
			Public Property IsCompleted() As Boolean
		End Class
	End Class
End Namespace
